spec/features/user_registration_spec.rb

```diff
@@ -0,0 +1,46 @@
+# /spec/features/user_registration_spec.rb
+require 'rails_helper'
+
+RSpec.feature 'New User Registration' do
```

Comment:
I like the "<entity> <does thing>" form for features. E.g. "Visitor registers" in this case. But it's just a matter of taste of course.